Sources/Twift.swift

```diff
@@ -106,6 +109,10 @@ public class Twift: NSObject, ObservableObject {
     var refreshedOAuthUser = try JSONDecoder().decode(OAuth2User.self, from: data)
     refreshedOAuthUser.clientId = clientId
     
-    self.authenticationType = .oauth2UserAuth(refreshedOAuthUser)
+	self.authenticationType = .oauth2UserAuth(refreshedOAuthUser)
+	self.oauthUser = refreshedOAuthUser
```

Comment:
This can be removed if `oauthUser` becomes a computed property.

(Also, can you check your indentation settings are 2 spaces for consistency? Thanks!)

Reply:
I was wondering why it keeps shifting and if I added an extra bracket 😅 Have defaulted to 4!